### DISCUSSION TITLE
Upgrade from CKEditor 4 to CKEditor 5 for DNNPlatform

### DISCUSSION THREAD

**Discussion by pcunningham222** [dnnsoftware/Dnn.Platform#6212] [General]
Hello everyone, good evening!  My colleagues and I are attempting to complete a trial upgrade to CKEditor 5 for one of our clients, whose application is running on DNN Platform 9.13.  We are finding it to be extremely difficult to create the custom HTML Editor Provider for CKEditor 5 and resolve all of the backwards compatibility issues that immediately manifest when we switched over to the new Provider.

Has anyone completed this process successfully?  If so, could I possibly impose on you to share as much as possible about what you learned during the process?  In particular, how long did it take you to complete the migration, and roughly how many work hours were expended in retrofitting the site assists for the new CKEditor 5 HTML Provider?  If anyone has an actual runbook documenting the steps they took to complete the migration, that would be so tremendously helpful.

Thank you very much in advance!

-Peter

**Comment by Timo-Breumelhof** [comment]
I don't know of anybody completing this TBH.
Simply upgrading the JS part of the provider will not work I guess as both editors are quite different.
As you might have read, unfortunately,  we can't include CKEditor 5 with DNN, see: https://github.com/dnnsoftware/Dnn.Platform/issues/5795

**Comment by Timo-Breumelhof** [comment]
FYI, you could also post this in the forums.
I think there some DNN users following those that will not see this discussion. https://dnncommunity.org/forums
